spec/unit/provider/package/windows/exe_package_spec.rb

```diff
@@ -34,7 +34,6 @@
     {
       'DisplayName'      => ['My App', ''],
       'UninstallString'  => ['E:\uninstall.exe', ''],
-      'SystemComponent'  => [nil, 1],
       'WindowsInstaller' => [nil, 1],
       'ParentKeyName'    => [nil, 'Uber Product'],
       'Security Update'  => [nil, 'KB890830'],
```

Comment:
Should we also remove the check if security update as some might want to do this as well.